ddp-workspace/projects/ddp-dsm-ui/src/app/field-datepicker/field-datepicker.component.html

```diff
@@ -1,11 +1,11 @@
 <span class="Display--Inline-Block">
   <b *ngIf="fieldName != null">{{fieldName}}</b>
   <mat-form-field class="DatePicker--Input">
-    <input matInput maxlength="10" [ngClass]="{'Input': true, 'Color--Field--Patched': colorDuringPatch}"
+    <input matInput maxlength="10" [ngClass]="{'Input': true, 'Color--Field--Patched': hasDateChanged}"
            placeholder="{{getDateFormatPlaceholder(dateFormat)}}"
```

Comment:
I would have a pure pipe (lowerecase) here instead of a function call.
Example:
`placeholder="{{dateFormat | lowercase}}"`

Reply:
> @@ -1,11 +1,11 @@ {{fieldName}} - 
{{getDateFormatPlaceholder(dateFormat)}}
 - if (this.datePicker != null) { - this.emitDate(Utils.getDateFormatted(this.datePicker, Utils.DATE_STRING)); - this.colorDuringPatch = false; - this.error = null; - } else { - this.error = this.dateFormat; - } - } else if (tmp === Utils.DATE_PARTIAL) { - this.error = null; - this.emitDate(Utils.getPartialDateFormatted(this._dateString, this.dateFormat)); + if (this._dateString === '') { + this.emitDate(''); + } else { + if (this._dateString !== 'N/A' && this._dateString !== 'Not Found') { + const tmp = Utils.parseDate(this._dateString, this.dateFormat, this.allowUnknownDay); + // console.log